### PR TITLE
Move footer information to a modal

### PR DESCRIPTION
**Pull request by nicksellen**
Infinite scroll and footers doesn't make much sense together and now we want a chat on the main page with the input box at the bottom, so seems to make sense to dump all this about karrot info elsewhere :)

Also included in https://github.com/yunity/karrot-frontend/pull/1291

### CODE REVIEW

**Comment by nicksellen** [comment]
Ah, I removed the `<UnsupportedBrowserWarning/>` but didn't add it back anywhere...

**Comment by tiltec** [comment]
Looks good!

I think you can remove `KFooter.vue` as it's not used.

What about `GLOBAL.FSWW_NOTE` ("Read our info page"), should we remove it from `locale-en.json`?

File: src/base/pages/MainLayout.vue [hunk 1]
```diff
@@ -60,6 +60,20 @@
           <SidenavTitle @click="toggleSidenav" />
           <RouterView name="sidenav" />
           <MobileSidenav/>
+          <QItem
+            link
+            @click.native="toggleAbout()"
+          >
+            <QItemSide class="text-center">
+              <KarrotLogo
+                disable-loading="true"
```
Comment (tiltec) [inline src/base/pages/MainLayout.vue]: > type check failed for prop "disableLoading". Expected Boolean, got String with value "true"

File: src/topbar/components/KTopbarUI.vue [hunk 2]
```diff
@@ -11,7 +11,10 @@
         :src="currentGroup.photoUrls.thumbnail"
         style="height: 95%"
       >
-      <KarrotLogo v-else />
+      <KarrotLogo
+        v-else
+        :show-loading="true"
```
Comment (tiltec) [inline src/topbar/components/KTopbarUI.vue]: Just `show-loading` should be enough!

Comment (nicksellen) [reply src/topbar/components/KTopbarUI.vue]: Ah yeah, I wasn't sure if that suddenly made it a directive... will update :)

**Pull request merged**